### PR TITLE
Deploy prow on app.ci

### PR DESCRIPTION
**Pull request by alvaroaleman**
This pr adds manifests and other configuration to deploy Prow on our new controlplane cluster, which is available at https://console-openshift-console.apps.ci.m4r3.p1.openshiftapps.com/ and which I took the freedom to name `app.ci`.

To be specific it:
* Adds a new `core-services-v2` directory with a `prow` subfolder that mirrors whats in `core-services/prow`, with some changes
* Adds a `core-services-v2/.diff` file that contains the expected diff to core-services and accompanying `verify` and `update` maketargets as well as a presubmit
* secret-bootstrapper config for a kubeconfig with access to the `app.ci` cluster, this is alreay rolled out
* A postsubmit that deploys to the `app.ci` cluster (No idea if it will work)

Both `horologium` and `plank` are initially deployed with zero replicas to make sure we don't create prowjob resources for periodics and to make sure we do not test pods in case something else I didn't think about creates prowjobs.

/assign stevekuznetsov petr-muller

### CODE REVIEW

File: core-services-v2/.diff [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+-    replicas: 1
++    replicas: 0
+     revisionHistoryLimit: 2
+     selector:
+       matchLabels:
+diff --recursive -u --new-file '--label=o' '--label=m' o m
+--- o
++++ m
+@@ -0,0 +1,51 @@
++kind: List
```
Comment (alvaroaleman) [inline core-services-v2/.diff]: We could probably add the namespaces to the current config as well

File: core-services-v2/prow/03_deployment/namespaces.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+kind: List
```
Comment (petr-muller) [inline core-services-v2/prow/03_deployment/namespaces.yaml]: Applyconfig goes over files in lexicographical order so we should make sure this one goes first.

**Comment by alvaroaleman** [comment]
> Can we have clusters/app.ci/prow instead of core-services-v2 and overall have app.ci instead of v2? I think we don't actually need that core/non-core taxonomy on this cluster at all...

Done

**Pull request merged**